From 65fa6f5917fb83fb3a638e34ce1f6be69f717be3 Mon Sep 17 00:00:00 2001
From: "Wong, Vincent Por Yin" <vincent.por.yin.wong@intel.com>
Date: Thu, 26 Dec 2019 11:04:59 +0800
Subject: [PATCH 16/17] REVERTME: net: stmmac: disable mdio interrupts for EHL
 & TGL

This workaround is required when running PHC2SYS and AF_XDP ZC at the
same time on EHL devices.

Signed-off-by: Wong, Vincent Por Yin <vincent.por.yin.wong@intel.com>
Signed-off-by: Voon Weifeng <weifeng.voon@intel.com>
---
 drivers/net/ethernet/stmicro/stmmac/hwif.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/ethernet/stmicro/stmmac/hwif.c b/drivers/net/ethernet/stmicro/stmmac/hwif.c
index 2baab98800e0..41c4f7f6e7fb 100644
--- a/drivers/net/ethernet/stmicro/stmmac/hwif.c
+++ b/drivers/net/ethernet/stmicro/stmmac/hwif.c
@@ -292,7 +292,7 @@ static int stmmac_dwmac4_quirks(struct stmmac_priv *priv)
 		.gmac = false,
 		.gmac4 = true,
 		.xgmac = false,
-		.mdio_intr_en = true,
+		.mdio_intr_en = false,
 		.has_serdes = true,
 		.tbs = true,
 		.min_id = DWMAC_CORE_5_10,
-- 
1.9.1

